src/utils/util/resolver-pipeline/select-or-custom-merge.xsl

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="3.0"
+    xmlns="http://csrc.nist.gov/ns/oscal/1.0"
+    xmlns:o="http://csrc.nist.gov/ns/oscal/1.0"
+    xmlns:opr="http://csrc.nist.gov/ns/oscal/profile-resolution"
+    xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns:math="http://www.w3.org/2005/xpath-functions/math"
+    exclude-result-prefixes="xs math o"
+    xpath-default-namespace="http://csrc.nist.gov/ns/oscal/1.0">
+
+    <xsl:include href="oscal-profile-resolve-functions.xsl"/>
+
+    <!-- A control is included if it is selected by the provided import instruction -->
+    <xsl:template match="control" mode="o:select" as="element(o:control)?">
+        <xsl:param name="import-instruction" tunnel="yes" required="yes"/>
+        <xsl:choose>
+            <xsl:when test="o:selects($import-instruction,.)">
+                <xsl:copy copy-namespaces="no">
+                    <xsl:call-template name="add-process-id"/>
+                    <xsl:apply-templates mode="#current" select="node() | @*"/>
+                </xsl:copy>
+           </xsl:when>
+           <xsl:otherwise>
+               <!-- Visit child controls in case they are selected using
+                   with-parent-controls="no". -->
+               <xsl:apply-templates mode="#current" select="control"/>
+           </xsl:otherwise>
```

Comment:
This wasn't in the original Selection phase code from which this file was created. Adding it makes with-parent-controls="no" work correctly in both the Selection and Merge phases.